l1-contracts/test/base/DecoderBase.sol

```diff
@@ -99,4 +75,158 @@ contract DecoderBase is TestBase {
   function max(uint256 a, uint256 b) internal pure returns (uint256) {
     return a > b ? a : b;
   }
+
+  function updateHeaderArchive(bytes memory _header, bytes32 _archive)
+    internal
+    pure
+    returns (bytes memory)
+  {
+    assembly {
+      mstore(add(_header, 0x20), _archive)
+    }
+    return _header;
+  }
+
```

Comment:
Thank you for adding these, I always hated editing the assembly for these use cases!!